workflows/sarek.nf

```diff
@@ -1000,7 +1000,7 @@ workflow SAREK {
         ch_versions = ch_versions.mix(TUMOR_ONLY_VARIANT_CALLING.out.versions)
 
         //QC
-        VCF_QC(vcf_to_annotate, intervals_bed_combined)
+        VCF_QC(vcf_to_annotate, TUMOR_ONLY_VARIANT_CALLING.out.manta_vcf_tbi, intervals_bed_combined)
```

Comment:
I know that command only works for the test for manta-tumor-only. I guess one would have to do something like the "mixed" `vcf_to_annotate` for the tbi-files.